nengo_spinnaker/operators/lif.py

```diff
@@ -135,14 +135,21 @@ def make_vertices(self, model, n_steps):  # TODO remove n_steps
         # can be quite lax to allow for lots of data probing.
         # TODO: Include other DTCM usage
         # TODO: Include CPU usage constraint
+        def cpu_usage(sl):
+            n_neurons = (sl.stop - sl.start)
+            return (245 + 43*self.ensemble.size_in + 100 + 702*size_out +
+                    188 + 69*n_neurons + 13*n_neurons*self.ensemble.size_in)
+
```

Comment:
A comment here about a resource for those numbers would be handy (perhaps a reference to the paper?)